docs/nodes/NODES.md

```diff
@@ -0,0 +1,68 @@
+# Using the Node Editor
+
+The nodes editor is a blank canvas allowing for the use of individual functions and image transformations to control the image generation workflow. The node processing flow is usually done from left (inputs) to right (outputs), but is not always the case the more complex the node graph becomes. Nodes inputs and outputs are connected by dragging connectors from node to node.
+
+To better understand how nodes are used, think of how an electric power bar works. It takes in one input (electricity from a wall outlet) and passes it to multiple devices through multiple outputs. Similarly, a node could have multiple inputs and outputs functioning at the same (or different) time, but all node outputs pass information onward like a power bar passes electricity. Not all outputs are compatible with all inputs, however - Each node has different constraints on how it is expecting to input/output information. In general, node outputs are colour-coded to match compatible inputs of other nodes.
+
+
+If you're not familiar with Diffusion, take a look at our [Diffusion Overview.](../help/diffusion.md) Understanding how diffusion works will enable you to more easily use the Nodes Editor and build workflows to suit your needs.
+
+## Important Concepts
+
+There are several node grouping concepts that can be examined with a narrow focus. These (and other) groupings can be pieced together to make up functional graph setups, and are important to understanding how groups of nodes work together as part of a whole. Note that the screenshots below aren't examples of complete functioning node graphs (see Examples).
+
+### Noise
+
+An initial noise tensor is necessary for the latent diffusion process. As a result, the Denoising node requires a noise node input.  
+
+![groupsnoise](../assets/nodes/groupsnoise.png)
+
+### Conditioning
+
+Conditioning is necessary for the latent diffusion process, whether empty or not. As a result, the Denoising node requires positive and negative conditioning inputs. Conditioning is reliant on a CLIP text encoder provided by the Model Loader node.
+
+![groupsconditioning](../assets/nodes/groupsconditioning.png)
+
+### Image Space & VAE
+
+The ImageToLatents node doesn't require a noise node input, but requires a VAE input to convert the image from image space into latent space. In reverse, the LatentsToImage node requires a VAE input to convert from latent space back into image space.
```

Comment:
Easier to describe these nodes functionally. ImageToLatents takes a pixel image and a VAE as inputs and outputs a latents. LatentsToImage reverses the process.

Reply:
Updated with this description